tests/ui-testing/playwright-tests/remotepipeline.spec.js

```
@@ -0,0 +1,532 @@
+};
+export const b64EncodeStandard = (str) => {
+  try {
+    return btoa(
+      encodeURIComponent(str).replace(/%([0-9A-F]{2})/g, function (match, p1) {
+        return String.fromCharCode(parseInt(`0x${p1}`));
+      })
+    );
+  } catch (e) {
+    console.log("Error: getBase64Encode: error while encoding.");
+  }
+};
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)